docs/Installation/Installation_Guide.rst

```diff
@@ -16,6 +16,6 @@ Kubernetes cluster project, as described in "`Setting Up a MetalK8s Cluster`_".
    topics/Setting_Up_a_Cluster
    topics/Installing_Zenko
    topics/configure_ingress
-
+   topics/Cosmos
```

Comment:
Maybe the title can be more generic? Like "Setting up an NFS" or "Adding NFS storage locations". I think people may just brush over the "Cosmos" topic because it's not known.

Reply:
Fair critique.